Makefile.am

```diff
@@ -88,7 +88,7 @@ CI_DOCKERFILE ?= $(srcdir)/dockerfile/anaconda-ci
 RPM_DOCKERFILE ?= $(srcdir)/dockerfile/anaconda-rpm
 CI_NAME ?= $(CONTAINER_REGISTRY)/rhinstaller/anaconda-ci
 RPM_NAME ?= $(CONTAINER_REGISTRY)/rhinstaller/anaconda-rpm
-CI_TAG := $(or $(CI_TAG),$(GIT_BRANCH))
+CI_TAG := $(or $(CI_TAG),$(shell echo "$(GIT_BRANCH)" | sed -e 's/^\(f[0123456789]\{2\}\).*/\1/'))
```

Comment:
I would like to have the `fXX` in a separate variable. It could be beneficial for future changes in the Makefile.

Reply:
Well, yes, except that is not going to be just f34. It's also master. But it makes sense to have this separately.